source/portable/mbedtls/core_pkcs11_mbedtls.c

```diff
@@ -239,6 +239,13 @@
  */
 #define PKCS11_USING_HMAC                  ( 1 )
 
+/**
+ * @brief Define the MBEDTLS_VERSION_NUMBER as 0 if it is not defined, done for MISRA rule 20.9
+ */
+#ifndef MBEDTLS_VERSION_NUMBER
+#define MBEDTLS_VERSION_NUMBER 0
```

Comment:
Fairly sure this should be indented - could be something wrong with your uncrustify?